CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 1.16.0
+
+* Update SonarPython 2.5 -> [2.10](https://jira.sonarsource.com/secure/ReleaseNote.jspa?projectId=10958&version=15721), 33 new rules, support for Python 3.8, improved accuracy through use of built-in types
```

Comment:
If think we should stop mentioning `SonarXXX` outside SonarSource. Please replace by something like: `Update Python analyzer`